internal/lib/lib.go

```diff
@@ -258,6 +258,13 @@ func GetNamespaceToSync() string {
 	return ""
 }
 
+func GetEnableRHI() bool {
```

Comment:
log the error to let the user know that we are defaulting this to false?

Reply:
Debug logged it.